docs/source/Appendix_B.md

```diff
@@ -271,7 +241,7 @@ for additional information on the tag extension rules.
 
 **a.**  A tag has `tagGroup` or `topLevelTagGroup` attribute, but is not enclosed in parentheses.   
 **b.**  A tag with the `topLevelTagGroup` does not appear at a HED tag group at the top level in an assembled HED annotation. 
-**c.**  Multiple tags with the `topLevelTagGroup` attribute appear in the same top-level tag group.
+**c.**  Multiple tags with the `topLevelTagGroup` attribute appear in the same top-level tag group.(note: Duration and Delay may be paired)

```

Comment:
**c.**  Multiple tags with the `topLevelTagGroup` attribute appear in the same top-level tag group. Note: a single `Duration` and a single `Delay` may be in the same group.